ProcessLib/SmallDeformationWithLIE/Common/Utils.cpp

```diff
@@ -0,0 +1,65 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ */
+
+#include "Utils.h"
+
+#include <cmath>
+
+#ifndef Q_MOC_RUN // to avoid Qt4 bug, https://bugreports.qt.io/browse/QTBUG-22829
+#include <boost/math/constants/constants.hpp>
+#endif
+
+#include "MeshLib/Elements/Element.h"
+#include "MeshLib/Node.h"
+
+namespace ProcessLib
+{
+namespace SmallDeformationWithLIE
+{
+
+namespace
+{
+
+void computeNormalVector2D(MeshLib::Element const& e, Eigen::Vector3d & normal_vector)
```

Comment:
Is the formula for lines only?
Since we faced some problems with the orientation of normal vectors recently: Is the normal vector oriented in the right direction, i.e., is it consistent with our current code? (Maybe have a small discussion about it?)